File: NetBeans_IDE/MmgGameApiJava/src/net/middlemind/MmgGameApiJava/MmgCore/LoadResourceUpdateHandler.java
package net.middlemind.MmgGameApiJava.MmgCore;

/**
 * Class to define the LoadDatUpdateHandler interface.
 * Created by Middlemind Games 01/22/2020
 * 
 * @author Victor G. Brusca
 */
public interface LoadResourceUpdateHandler {
    
    /**
     * The message handling method to override.
     * 
     * @param obj       A LoadDatUpdateMessage.
     */
    public void HandleUpdate(LoadResourceUpdateMessage obj);   
}

File: VisualStudio_IDE/MmgGameApiCs/MmgGameApiCs/src/net/middlemind/MmgGameApiCs/MmgCore/LoadResourceUpdateHandler.cs
using System;

namespace net.middlemind.MmgGameApiCs.MmgCore
{
    /// <summary>
    /// Class to define the LoadDatUpdateHandler interface.
    /// Created by Middlemind Games 01/22/2020
    ///
    /// @author Victor G.Brusca
    /// </summary>
    [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "<Pending>")]
    public interface LoadResourceUpdateHandler
    {
        /// <summary>
        /// The message handling method to override.
        /// </summary>
        /// <param name="obj">A LoadDatUpdateMessage.</param>
        public void HandleUpdate(LoadResourceUpdateMessage obj);
    }
}